server/lib/scene/scene.get.js

```diff
@@ -26,31 +27,57 @@ async function get(options) {
     attributes: optionsWithDefault.fields,
     offset: optionsWithDefault.skip,
     order: [[optionsWithDefault.order_by, optionsWithDefault.order_dir]],
+    include: [
+      {
+        model: db.TagScene,
+        as: 'tags',
+        attributes: ['name'],
+      },
+    ],
   };
 
   if (optionsWithDefault.take !== undefined) {
     queryParams.limit = optionsWithDefault.take;
   }
 
+  // search by device feature selectors
+  if (optionsWithDefault.selectors) {
+    queryParams.where = {
+      [Op.or]: optionsWithDefault.selectors.split(',').map((selector) => ({
+        selector,
+      })),
+    };
+  }
   if (optionsWithDefault.search) {
-    queryParams.where = Sequelize.where(Sequelize.fn('lower', Sequelize.col('name')), {
+    queryParams.where = Sequelize.where(Sequelize.fn('lower', Sequelize.col('t_scene.name')), {
       [Op.like]: `%${optionsWithDefault.search}%`,
     });
   }
 
-  // search by device feature selectors
-  if (optionsWithDefault.selectors) {
+  if (optionsWithDefault.searchTags) {
+    const databaseTags = await Promise.all(
+      optionsWithDefault.searchTags.split(',').map((tag) => {
+        return db.TagScene.findAll({
+          attributes: ['scene_id'],
+          where: {
+            name: tag,
+          },
+        });
+      }),
+    );
+    const sceneIds = databaseTags.map((tags) => tags.map((tag) => tag.get({ plain: true }).scene_id));
+    const intersec = intersection(...sceneIds);
+
     queryParams.where = {
-      [Op.or]: optionsWithDefault.selectors.split(',').map((selector) => ({
-        selector,
+      [Op.or]: intersec.map((sceneId) => ({
```

Comment:
This override all previous search params, so in the UI, I typed a search + added a tag, and it doesn't take into account the text search